lib/tasks/pipeline_monitor.rake

```diff
@@ -296,18 +296,15 @@ class CheckPipelineRuns
       t_iter_start = t_now
       pr_ids = PipelineRun.in_progress.pluck(:id)
       pt_ids = PhyloTree.in_progress.pluck(:id)
-      pool = Thread.pool(THREAD_COUNT)
-      pr_ids.each do |prid|
-        pool.process do
-          update_pr(prid)
-        end
+      Parallel.each(pr_ids, in_processes: PROCESS_COUNT) do |prid|
+        @reconnected ||= ActiveRecord::Base.connection.reconnect! || true
```

Comment:
Sorry, what is the `|| true` supposed to do?

Reply:
This makes it only attempt reconnect once if it fails I believe it's from the docs. I will switch back to the old way which just has the reconnect, it worked before.